### PR TITLE
ENG-8772: Removing the ability to specify default limit size across a…

### PR DESCRIPTION
**Pull request by manjujames**
…ll disks.

Added hostname also to 'resource limit exceeded' error message.

### CODE REVIEW

File: src/frontend/org/voltdb/DiskResourceChecker.java [hunk 1]
```diff
@@ -106,10 +107,8 @@ private void findDiskLimitConfiguration(SystemSettingsType systemSettings, Paths
             return;
         }
 
-        String defaultSize = diskLimitConfig.getSize();
         List<DiskLimitType.Feature> features = diskLimitConfig.getFeature();
-        //If defaultsize is configured, that should be set as the limit size for all features not explicitly configured
-        if ( (features==null || features.isEmpty()) && (defaultSize == null || defaultSize.trim().isEmpty()) ) {
+        if (features==null || features.isEmpty()) {
```
Comment (akhanzode) [inline src/frontend/org/voltdb/DiskResourceChecker.java]: Warn here that no features are configured? Or will <disklimit /> will just get caught as No resource limit configured message? If yes ignore this comment.

Comment (manjujames) [inline src/frontend/org/voltdb/DiskResourceChecker.java]: Yes, I should add minOccurs with 1. Thanks.

**Comment by akhanzode** [comment]
Can you test with this change VMC can update anything else? like edit query timeout and save and see if it works. Rest looks good merge when blue.

File: tests/frontend/org/voltdb/TestJSONInterface.java [hunk 2]
```diff
@@ -1563,6 +1565,71 @@ public void testUpdateDeployment() throws Exception {
...
+            ss.setResourcemonitor(resourceMonitor);
+            String ndeptype = mapper.writeValueAsString(deptype);
+            params.put("deployment", ndeptype);
+            String pdep = postUrlOverJSON("http://localhost:8095/deployment/", null, null, null, 200, "application/json", params);
+            assertTrue(pdep.contains("Deployment Updated"));
+
+            memLimit.setSize("90%25");
+            ndeptype = mapper.writeValueAsString(deptype);
+            params.put("deployment", ndeptype);
+            pdep = postUrlOverJSON("http://localhost:8095/deployment/", null, null, null, 200, "application/json", params);
```
Comment (akhanzode) [inline tests/frontend/org/voltdb/TestJSONInterface.java]: Can you add a GET and check json that the values you updated are in deployment.

**Comment by akhanzode** [comment]
Looks good to me if jenkins is blue push.

**Pull request merged**